test/timeunit.test.ts

```diff
@@ -1,200 +1,201 @@
-import {assert} from 'chai';
 import {containsTimeUnit, convert, fieldExpr, formatExpression, TimeUnit} from '../src/timeunit';
 
 describe('timeUnit', () => {
   describe('containsTimeUnit', () => {
     it('should return true for quarter given quarter', () => {
       const fullTimeUnit = TimeUnit.QUARTER;
       const timeUnit = TimeUnit.QUARTER;
-      assert.equal(containsTimeUnit(fullTimeUnit, timeUnit), true);
+      expect(containsTimeUnit(fullTimeUnit, timeUnit)).toBeTruthy();
```

Comment:
Why truthy, not true? (Shouldn't this method return boolean?)

Reply:
Yeah, you're right. I fixed it.